script/hassfest/coverage.py

```diff
@@ -81,6 +81,15 @@ def validate(integrations: dict[str, Integration], config: Config) -> None:
 
             integration = integrations[integration_path.name]
 
+            if (
+                path.parts[-1] == "*"
+                and Path(f"tests/components/{integration.domain}/__init__.py").exists()
+            ):
```

Comment:
I think that if `tests/components/xxx/__init__.py` file exists, then it implies there is at least an attempt to add tests and we should encourage further coverage.